docs/java-rest/high-level/apis/indices_aliases.asciidoc

```diff
@@ -0,0 +1,85 @@
+[[java-rest-high-indices-aliases]]
+=== Update Aliases API
+
+[[java-rest-high-indices-aliases-request]]
+==== Index Aliases Request
+
+The Update Aliases API allows aliasing an index with a name, with all APIs
+automatically converting the alias name to the actual index name.
+
+An `IndicesAliasesRequest` must have at least one `AliasActions`:
+
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests}/IndicesClientDocumentationIT.java[indices-aliases-request]
+--------------------------------------------------
+<1> Creates an `IndicesAliasesRequest`
+<2> Creates an `AliasActions` that aliases index `test1` with `alias1`
+<3> Add the alias action to the request
+
+The following action types are supported: `add` - alias an index, `remove` - 
+remove the alias assosiated with the index and `remove_index` - deletes the 
+index. 
+
+Defining an alias with filters provide an easy way to create different 
+"views" of the same index. The filter can be defined using Query DSL and is 
+applied to all Search, Count, Delete By Query and More Like This operations 
+with this alias.
```

Comment:
here can we just describe how to add a filter rather than describing what filters do and where they get applied?